scripts/build-and-optimize.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+RUSTFLAGS="-C link-arg=-s ${RUSTFLAGS}"
+
+rm -rf "/target/"
+
+mkdir "/target/"
+
+rm -rf "/artifacts/"*
+
+for contract in $(echo "/code/contracts/"* | sed 's/ /\n/g' | sort)
```

Comment:
not sure `$(echo "/code/contracts/"* | sed 's/ /\n/g' | sort)`  would iterate over dirs ... what about `echo` -> `ls`?

Reply:
I tested it out, in the shell, through the container and CI. ;) I can of course change it to `ls` too.

Reply:
me too, the output in the shell gave only
`❯ for c in $(echo "/code/contracts/"* | sed 's/ /\n/g' | sort)
∙ do
∙ echo $c
∙ done
/code/contracts/*
`